frontend-auth-module/projects/sl-auth-app/src/app/app-routing.module.ts

```diff
@@ -1,10 +1,13 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
-
-const routes: Routes = [];
+import { HomeComponent } from './components/home/home.component';
+import { AuthGuard } from 'projects/sl-auth-lib/src/public-api';
+const routes: Routes = [
+  {path:'home', component:HomeComponent, canActivate: [AuthGuard] }
+];
 
 @NgModule({
-  imports: [RouterModule.forRoot(routes)],
+  imports: [RouterModule.forChild(routes)],
```

Comment:
Why was forChild added instead of forRoot? what advantage is it giving us, can you please explain?